pkg/projects/provider.go

```diff
@@ -60,96 +59,30 @@ type ProjectClient struct {
 func buildResty(URL string) (*resty.Client, error) {
 
 	u, err := url.ParseRequestURI(URL)
-
 	if err != nil {
 		return nil, err
 	}
+
 	baseUrl := fmt.Sprintf("%s://%s", u.Scheme, u.Host)
 	restyBase := resty.New().SetHostURL(baseUrl).OnAfterResponse(func(client *resty.Client, response *resty.Response) error {
 		if response == nil {
 			return fmt.Errorf("no response found")
 		}
+
 		if response.StatusCode() >= http.StatusBadRequest {
 			return fmt.Errorf("\n%d %s %s\n%s", response.StatusCode(), response.Request.Method, response.Request.URL, string(response.Body()[:]))
 		}
+
 		return nil
 	}).
 		SetHeader("content-type", "application/json").
 		SetHeader("accept", "*/*").
 		SetHeader("user-agent", "jfrog/terraform-provider-projects:"+Version).
 		SetRetryCount(5)
-	restyBase.DisableWarn = true
```

Comment:
You don't want to turn this off. You'll get a lot of noise testing against non https

Reply:
@chb0github Not sure how that gets in. I'll revert this.

Reply:
@chb0github the line is still here at https://github.com/jfrog/terraform-provider-projects/pull/4/files/986c481a9cc639ca3fc180daa5b88f612a0e6c84#diff-5f2b9c4a353c9b47955c5cf3a3caf4ef7c831efbd60c9b4b70cd19cea5937b10R83